### ISSUE TITLE
Valleyix dbsetup requires bookmark links to formats we don't have

### ISSUE THREAD

**Issue by abalmos** [OADA/oada-api-server#6]
`vriPrescription` is in the example for `application/vnd.valleyix.machine.1+json` but we do not have a format for that. Currently the dbsetup is configured to override the key's value with an empty object. However, any empty object at `vriPrescription` is a violation of `application/vnd.valleyix.machine.1+json`'s schema.

I think in this case the dbsetup should just remove the `vriPrescription` key.

**Comment by aultac** [comment]
agreed, will remove vriPrescription from dbsetup when I get back to this
